tests/PKSim.Tests/Core/SimulationConfigurationValidatorSpecs.cs

```diff
@@ -44,7 +44,7 @@ protected override void Context()
          _protocolToProtocolSchemaItemMapper = A.Fake<IProtocolToSchemaItemsMapper>();
          sut = new SimulationConfigurationValidator(_protocolToProtocolSchemaItemMapper);
 
-         _speciesPopulation.IsBodySurfaceAreaDependent = false;
+         _speciesPopulation.IsHeightDependent = false;
```

Comment:
@Yuri05 This is why I did not like it. The tests are not as expressive. But whatever